pkgs/development/python-modules/hypothesis/default.nix

```diff
@@ -27,14 +27,10 @@ buildPythonPackage rec {
   checkInputs = [ pytest pytest_xdist flaky mock pexpect ];
   inherit doCheck;
 
-  checkPhase = ''
-    rm tox.ini # This file changes how py.test runs and breaks it
-    py.test tests/cover
-  '';
```

Comment:
This reverts back to using `setuptoolsCheckPhase` which we are trying to avoid #71897.
I think you should try using `pytestCheckHook` instead.

Reply:
Ah, ok.
And when trying to use `pytestCheckHook`, many tests are failing...
So I put this `checkPhase` back.